metacat-common-server/src/main/java/com/facebook/presto/spi/ColumnDetailMetadata.java

```diff
@@ -70,4 +102,29 @@ public Boolean getIsSortKey() {
     public Boolean getIsIndexKey() {
         return isIndexKey;
     }
+
+    @Override
+    public boolean equals(final Object o) {
```

Comment:
Any reason to manually specify this equals and toString instead of using the Lombok annotation?